<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Bornaviridae-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Bornaviridae-GLUE</h1>
      <h2 class="project-tagline">A sequence data resource for bornaviruses.</h2>
      <a href="../index.html" class="btn">Home</a>
      <a href="./project.html" class="btn">Viruses</a>
      <a href="https://github.com/giffordlabcvr/Bornaviridae-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Bornaviridae-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>

    <section class="main-content">

	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Endogenous bornavirus (EVE) data</strong>
	   </h3>

		<p>
          We have used GLUE to organise the fossil record of bornaviruses.
          You can access these data
          <a href="https://github.com/giffordlabcvr/Bornaviridae-GLUE/tree/master">in this repository</a>. 
          This website provides quick links to specific data items. 

		</p>


	   <h3>
		 <a id="EVE reference sequences and data" class="anchor" href="#Ref_ecv_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>EVE reference sequences and data</strong>
	   </h3>

		<p>
           We constructed reference sequences for EVEs using alignments of EVE
           sequences derived from the same initial germline colonisation event - i.e.
           orthologous elements in distinct species, and paralogous 
           elements that have arisen via intragenomic duplication of EVE sequences.
  		</p>

		<p>
           Reference sequence data in tabular format are <a href="https://github.com/giffordlabcvr/Bornaviridae-GLUE/blob/master/tabular/eve/ehbv-refseqs-side-data.tsv"><strong>here</strong></a>.
  		</p>


	   <h3>
		 <a id="Raw EVE sequences and data" class="anchor" href="#Raw_digs_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Raw EVE sequences and data</strong>
	   </h3>

		<p>
          These are the raw data generated by
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">database-integrated genome screening (DIGS)</a>.
          The tabular files contain information about the genomic location of each EVE. 
          EVEs were classified by comparison to a polypeptide sequence reference library
          designed to represent the known diversity of bornaviruses - this includes extinct 
          lineages represented only by endogenous viral elements (EVEs).
 		</p>

		<p>
          These data were obtained via
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">DIGS</a>
          performed in vertebrate genome assemblies downloaded from 
          <a href="https://www.ncbi.nlm.nih.gov/genome/browse ">NCBI genomes</a>
          (2020-07-15).
		</p>


		<p>
           Raw data in tabular format are <a href="https://github.com/giffordlabcvr/Bornaviridae-GLUE/blob/master/tabular/eve/eve-side-data.tsv"><strong>here</strong></a>.
  		</p>

		<p>
           Nucleotide level data in FASTA format (individual files) are <a href="https://github.com/giffordlabcvr/Bornaviridae-GLUE/tree/master/sources/fasta-digs-eve"><strong>here</strong></a>.
  		</p>
		



	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>


		<p>
		

		Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		GLUE: A flexible software system for virus sequence data.
		<br>
		<i>BMC Bioinformatics</i>
		<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		<br>
		<br>

		Zhu H, Dennis T, Hughes J, and RJ Gifford
		<b>(2018)</b>
		<br>
		Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		<a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		<br>
		<br>

		Katzourakis A. and RJ. Gifford
		<b>(2010)</b>
		<br>
		Endogenous viral elements in animal genomes.
		<br>
		<i>PLoS Genetics</i>
		<a href="http://www.plosgenetics.org/article/info:doi/10.1371/journal.pgen.1001191" target="_blank">[view]</a>

 		</p>


	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Bornaviridae-GLUE/">Bornaviridae-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



